deeppavlov/core/common/chainer.py

```diff
@@ -192,3 +192,4 @@ def destroy(self):
         for in_params, out_params, component in self.pipe:
             if inspect.ismethod(getattr(component, 'destroy', None)):
                 component.destroy()
+            del component
```

Comment:
This does nothing. `component` variable is deleted in every iteration of this `for`. And the last one is deleted after leaving the `destroy` method

Reply:
Why `component` variable is deleted without `del component`?